test/functional/fixtures/api/es-next/take-screenshot/test.js

```diff
@@ -1,14 +1,19 @@
-var expect                  = require('chai').expect;
-var config                  = require('../../../../config.js');
-var checkScreenshotsCreated = require('../../../../assertion-helper.js').checkScreenshotsCreated;
+var expect                        = require('chai').expect;
+var config                        = require('../../../../config.js');
+var checkScreenshotsCreated       = require('../../../../assertion-helper.js').checkScreenshotsCreated;
+var checkScreenshotsAreNotCreated = require('../../../../assertion-helper.js').checkScreenshotsAreNotCreated;
 
 
+var SCREENSHOT_PATH_MESSAGE_TEXT          = '___test-screenshots___';
+var SCREENSHOT_PATH_DO_NOT_SETTED_MESSAGE = '[cannot take screenshots because the screenshot directory is not specified]';
```

Comment:
`SCREENSHOT_PATH_NOT_SET_MESSAGE`
or - don't we use DIR now?
`SCREENSHOT_DIR_NOT_SET_MESSAGE`